themes/src/main/resources/theme/keycloak-preview/account/resources/content.json

```diff
@@ -56,5 +56,14 @@
     "modulePath": "/content/my-resources-page/MyResourcesPage.js",
     "componentName": "MyResourcesPage",
     "hidden": "!features.isMyResourcesEnabled"
+  },
+  {
+    "id": "deleteAccount",
+    "path": "deleteAccount",
+    "label": "Delete Account",
```

Comment:
The label should be a message bundle key instead of the English phrase "Delete Account"

Reply:
done!